pinc/graph_data.inc

```diff
@@ -68,7 +68,7 @@ function user_logging_on($past, $preceding)
     switch ($past) {
       case 'year':
           $min_timestamp = time() - 366 * $seconds_per_day;
-          $date_format = '%Y-%c-%d';
```

Comment:
This ensures the month is 2 digits (02 instead of 2), which apparently Safari does not like when doing a new Date('2020-2-08') for example.